torch/testing/_internal/common_methods_invocations.py

```diff
@@ -10244,6 +10282,25 @@ def ref_pairwise_distance(input1, input2):
            ),
            supports_forward_ad=False,
            supports_out=False),
+    OpInfo('nn.functional.binary_cross_entropy',
+           aten_name='binary_cross_entropy',
+           sample_inputs_func=sample_inputs_binary_cross_entropy,
+           dtypesIfCPU=floating_types(),
+           dtypesIfCUDA=floating_types_and(torch.float16, torch.bfloat16),
+           supports_out=False,
+           skips=(
+               # FIXME: from derivatives.yaml:
+               # not_implemented("binary_cross_entropy_backward wrt `target`")
+               DecorateInfo(
+                   unittest.skip('Skipped!'), 'TestGradients',
+                   'test_fn_gradgrad'),
```

Comment:
I take it there's no way to skip validating gradients wrt a single input?

Beyond the scope of this PR, but it seems non-ideal to lose validation of gradgradcheck wrt the input just because this isn't implemented for the target.

Reply:
can't comment on this since i don't know. so waiting for someone else to chime in

Reply:
Joel's commnet is on point, but alas, we cannot do much about it at the moment.
@nkaretnikov might revisit this operation in the future as part of his onboarding, and we may consider to add all these missing features to this operation.